src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -258,7 +258,7 @@ class PeerActor[R <: HandshakeResult](
         handleTerminated(rlpxConnection, 0, Handshaked) orElse {
 
           case RLPxConnectionHandler.MessageReceived(message) =>
-            log.debug(s"Received message: {} from $peerId", message)
+            log.debug(s"Received message: {} from $peerId", message.getClass.getName)
```

Comment:
Noooooo, this logs were useful on tesnet debugging

Reply:
And disruptive when running regular sync on mordor with debug level ;)

Reply:
Totally, I usually remove this logs when local testing xD maybe our cluster should eventually have a different logback than other nodes?

But given that we don't have that in place and that this logs are useful on the testnet we'll have to stick with this disruptive logs, we can create a future task to do in conjunction with devops to bring a separate logback to the cluster or thing of another alternative

Reply:
I see, restoring then